<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbSettingsDialogBase</class>
 <widget class="QDialog" name="DbSettingsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DbSettings</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>100</y>
     <width>127</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>cleaned</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>20</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>database</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>60</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>schema</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dbCombo">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>18</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>100</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>table name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="schemaCombo">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>58</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
